modopt/signal/svd.py

```diff
@@ -85,7 +85,7 @@ def calculate_svd(data):
     if (not isinstance(data, np.ndarray)) or (data.ndim != 2):
         raise TypeError('Input data must be a 2D np.ndarray.')
 
-    return svd(data, check_finite=False, lapack_driver='gesvd')
+    return svd(data, check_finite=False, lapack_driver='gesvd', full_matrices=False)
```

Comment:
Please update to preserve PEP8 standards. Line is over 79 characters long.

```python
return svd(data, check_finite=False, lapack_driver='gesvd', 
           full_matrices=False)
```

Reply:
Also note that `diagsvd` is no longer needed from import.

```python
from scipy.linalg import svd, diagsvd
```

Finally, you will need to update the expected output for the `test_calculate_svd` unit test. 

```python
self.data2 = (np.array([[-0.01744594, -0.61438865],
                        [-0.08435304, -0.50397984],
                        [-0.15126014, -0.39357102],
                        [-0.21816724, -0.28316221],
                        [-0.28507434, -0.17275339],
                        [-0.35198144, -0.06234457],
                        [-0.41888854,  0.04806424],
                        [-0.48579564,  0.15847306],
                        [-0.55270274,  0.26888188]]),
              np.array([42.23492742, 1.10041151]),
              np.array([[-0.67608034, -0.73682791],
                        [0.73682791, -0.67608034]]))
```